.github/workflows/ci.yml

```diff
@@ -37,6 +37,7 @@ jobs:
     - name: Install Python dependencies
       run: |
         python -m pip install --upgrade pip
+        pip install .[dev]
         pip install coverage
```

Comment:
This could be moved to the `[dev]` dependencies and then installed via the command above. 

Same with the dependencies in line 66 and 87.